src/thermo/PlasmaPhase.cpp

```diff
@@ -564,9 +572,14 @@ double PlasmaPhase::elasticPowerLoss()
         rate += concentration(m_targetSpeciesIndices[i]) *
                 m_elasticElectronEnergyLossCoefficients[i];
     }
+    const double q_elastic = Avogadro * Avogadro * ElectronCharge * concentration(m_electronSpeciesIndex) * rate;
```

Comment:
Please try to adhere to our style recommendation of 88 characters per line maximum.